install/installer/cluster_dev.py

```diff
@@ -576,7 +576,11 @@ def create_aws_user_and_permissions(
     # AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
     try:  # Required "iam:ListUsers"???, "iam:ListAccessKeys"
         response = iam.list_access_keys(UserName=user)
-
+    except iam.exceptions.NoSuchEntityException:  # User not yet exist
+        pass
```

Comment:
It's more useful to have debug logging (which isn't printed out in your current setup but may prove useful whenever you need to go deeper):
```suggestion
        logger.debug('User "%s" does not exist yet')
```